assets/eip-5496/test/test.js

```diff
@@ -19,7 +19,7 @@ contract("ERC5496", async accounts => {
         await demoContract.increasePrivileges(false);

```

Comment:
File comment: consider add a test for EIP-165 identifier.

Reply:
We added a new test case for EIP-165.